BLSQ/iaso
# IA-2915: Add descriptions of mobile feature flags on hover

Explain what problem this PR is resolving
- Add descriptions of mobile feature flags on hover
Related JIRA tickets : [IA-2915](https://bluesquare.atlassian.net/browse/IA-2915)

## Self proofreading checklist


## Doc
- Tell us where the doc can be found (docs folder, wiki, in the code...)

## Changes
- Add renderOption to render with a tooltip on options not selected
- Add renderTags to render with a tooltip on already selected options

## How to test
- Go in admin -> project
- Edit or create a project
- Check if on each featureflag there is a tooltip message

## Print screen / video
[Screencast from 2024-05-03 15-37-12.webm](https://github.com/BLSQ/iaso/assets/19631540/edad6bd9-75c3-4097-a2fd-26b0cb15cda6)


## Notes

Things that the reviewers should know: known bugs that are out of the scope of the PR, other trade-offs that were made.
If the PR depends on a PR in bluesquare-components, or merges into another PR (i.o. main), it should also be mentioned here


[IA-2915]: https://bluesquare.atlassian.net/browse/IA-2915?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
